src/parser/statement.js

```diff
@@ -98,6 +98,18 @@ pp.parseStatement = function (declaration, topLevel) {
     case tt.semi: return this.parseEmptyStatement(node);
     case tt._export:
     case tt._import:
+      if (this.hasPlugin("importFunctions")) {
+        // peek ahead and see if next token is a starting paren
+        let state = this.state.clone();
+        this.next();
+        if (this.match(tt.parenL)) {
+          this.state = state;
+          break;
+        } else {
+          this.state = state;
+        }
```

Comment:
I _think_ this can be simplified to this, though I'm not an expert on Babylon.

    if (this.hasPlugin("importFunctions") && this.lookahead().type === tt.parenL) break;